README.md

```diff
@@ -0,0 +1,75 @@
+Stitch
```

Comment:
This file is needed only for forked repo main page - to have new (temporal?) install instructions and a link to NEW.md. After pull request completed it won't be needed.